noxfile.py

```diff
@@ -124,10 +134,24 @@ def coverage_release(session: nox.Session):
         *posargs,
         "-e",
         f"antsibull_build_command={build_command!r}",
+        env={"ANSIBLE_COLLECTIONS_PATH": str(collections), **env},
     )
-    session.run("coverage", "combine", *Path(".").glob(".coverage.*"))
-    session.run("coverage", "report")
-    session.run("coverage", "xml", "-i")
+
+    combined = map(str, tmp.glob(".coverage.*"))
+    session.run("coverage", "combine", *combined, env=env)
+    session.run("coverage", "report", env=env)
+
+
+@nox.session
+def coverage(session: nox.Session):
```

Comment:
We should mention the coverage session the README. You can take the text from antsibull-changelog's README.

Reply:
598dd4a6381b11c5e671d7a2f7d96c77323a5b44 - I modified the text a bit (PR for antsibull-changelog will come up as well).

Reply:
Ref: https://github.com/ansible-community/antsibull-changelog/pull/114

Reply:
Ok, 92064f692486b1a2794ec92602c295ef7bfe2236 really fixes this :)